app/model.py

```diff
@@ -19,7 +19,12 @@ def sort_mentors(mentor: Dict) -> Tuple:
             )
 
         results = sorted(
-            self.db.search("Mentors", mentee["tech_stack"]),
+            self.db.read("Mentors", {
+                "tech_stack": mentee["tech_stack"],
+                "is_active": True,
+                "accepting_new_mentees": True,
+                "validate_status": 'approved'
+                }),
```

Comment:
One tiny PEP8 issue here on line 27, unindent once to make it aligned with the opening brackets on line 22.

Reply:
Or Command-Option-L to make PyCharm do it for you.

Reply:
this is actually a sick burn since Steven is still on vscode...